test/dialogflow-app-test.js

```diff
@@ -41,6 +41,7 @@ const {
   headerV2,
   MockResponse,
   MockRequest,
+  fakeConversationId,
```

Comment:
I don't see the `mocking.js` file changed. What is this importing?

Since this seems to be a dialogflow specific change and the `fakeConversationId` is not shared across other files, for now, you should just create it where it's needed (in this case just the `#getConversationId` test)

Reply:
There is already fakeConversationId in `mocking.js` file -> https://github.com/actions-on-google/actions-on-google-nodejs/blob/master/test/utils/mocking.js#L293

I think it's used in other test(s) so I wanted to reuse it here.

Reply:
Oh I see. Yeah, this is fine then.